voltixApp/VoltixApp/View Models/SendCryptoViewModel.swift

```diff
@@ -224,48 +224,13 @@ class SendCryptoViewModel: ObservableObject, TransferViewModel {
             return isValidForm
         }
         
-        let coinName = tx.coin.chain.name.lowercased()
-        let key: String = "\(tx.fromAddress)-\(coinName)"
-        
-        if  tx.coin.chain.chainType == ChainType.UTXO {
-            let walletBalanceInSats = utxo.blockchairData.get(key)?.address?.balance ?? 0
-            let totalTransactionCostInSats = tx.amountInSats + BigInt(tx.feeInSats)
-            print("Total transaction cost: \(totalTransactionCostInSats)")
+        if tx.isAmountExceeded && tx.coin.isNativeToken {
```

Comment:
a few questions about this:
1. What happens if the tx.coin.isNativeToken is false, for example ,ERC20/BEP20 etc , do we also check their send amount is larger than balance
2.  What if user trying to send ERC20 , but they don't have ETH available?

Reply:
1 - ERC20 uses Native Tokens (Coins) to pay the fees. So they always can send the full amount of ERC20 Token.
2 - Then we will have an error of not enough fees. We will need to raise another PR to show the error in the tx complete screen.

Reply:
1 - ERC20 uses Native Tokens (Coins) to pay the fees. So they always can send the full amount of ERC20 Token.
2 - We are now validating if they have native tokens to pay the fees